client/src/pages/calculator.js

```diff
@@ -33,7 +41,24 @@ const Calculator = () => {
 
     function patientList() {
         return records.map((record) => {
-                return <DropdownOption record = {record}/>
+                return <PatientDropdownOption record = {record}/>
+            }
+        );
+    }
+
+    // DropdownOption object; updates selected patient using initials when clicked
+    const CalculationDropdownOption = (props) => (
```

Comment:
I like the inclusion of the calc dropdown, this will be helpful when we have our larger list of calculations. The abstraction look robust.